aci/aci-hyperkube/patches/0001-Make-rkt-runtime-event-aggregable.patch

```diff
@@ -0,0 +1,34 @@
+From bc7a4a0f2c0e79d467a72c5e72553dee15e495a5 Mon Sep 17 00:00:00 2001
+From: Olivier Biesmans <olivier.biesmans@blablacar.com>
+Date: Wed, 8 Nov 2017 10:59:15 +0100
+Subject: [PATCH] Make rkt runtime event aggregable
+
+---
+ pkg/kubelet/rkt/rkt.go | 8 ++++----
+ 1 file changed, 4 insertions(+), 4 deletions(-)
+
+diff --git a/pkg/kubelet/rkt/rkt.go b/pkg/kubelet/rkt/rkt.go
+index d16c92db31..8f0c8a1cf0 100644
+--- a/pkg/kubelet/rkt/rkt.go
++++ b/pkg/kubelet/rkt/rkt.go
+@@ -1271,13 +1271,13 @@ func (r *Runtime) generateEvents(runtimePod *kubecontainer.Pod, reason string, f
+ 		uuid := utilstrings.ShortenString(id.uuid, 8)
+ 		switch reason {
+ 		case "Created":
+-			r.recorder.Eventf(events.ToObjectReference(ref), v1.EventTypeNormal, events.CreatedContainer, "Created with rkt id %v", uuid)
++			r.recorder.Eventf(events.ToObjectReference(ref), v1.EventTypeNormal, events.CreatedContainer, "Created Container")
+ 		case "Started":
+-			r.recorder.Eventf(events.ToObjectReference(ref), v1.EventTypeNormal, events.StartedContainer, "Started with rkt id %v", uuid)
++			r.recorder.Eventf(events.ToObjectReference(ref), v1.EventTypeNormal, events.StartedContainer, "Started Container")
+ 		case "Failed":
+-			r.recorder.Eventf(events.ToObjectReference(ref), v1.EventTypeWarning, events.FailedToStartContainer, "Failed to start with rkt id %v with error %v", uuid, failure)
++			r.recorder.Eventf(events.ToObjectReference(ref), v1.EventTypeWarning, events.FailedToStartContainer, "Start Container Failed")
```

Comment:
Failure should be the same most of the time.
Can you keep it ?

Reply:
Phrasing could be better too. `Failed to start container with error %v", failure`